src/cf-typography/usage.md

```diff
@@ -573,10 +583,10 @@ legal copy.
 
 ### Pull quote
 
-Use a pull quote to highlight excerpts from the current work.
-This is not to be confused with `blockquote` which quotes from an external work.
-Since a pull quote is an excerpt and repeats content from the article
-you should use the `aside` element.
+Use a pull quote to highlight excerpts from the current work. This is not to
+be confused with `blockquote` which quotes from an external work. Since a pull
+quote is an excerpt and repeats content from the article you should use the
+`aside` element.
```

Comment:
L:577 was longer than 80, same comment as before about not seeing a point in semantic breaks if they don't utilize the space given.